app/src/main/java/com/example/gradeassist/service/QuestionServiceStub.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.gradeassist.service
+
+import androidx.lifecycle.MutableLiveData
+import com.example.gradeassist.dto.Question
+
+class QuestionServiceStub : IQuestionService {
+    override fun fetchQuestionById(questionId: Int): MutableLiveData<Question> {
+        return MutableLiveData<Question>()
+    }
+}
```

Comment:
Created QuestionServiceStub because there was a ScoreServiceStub. (not really sure if necessary)

Reply:
If you don't use it, it is clutter and should be left out.  Clutter is technical debt.

Reply:
Understood, thanks for the details.